service/commentutil/commentutil.go

```diff
@@ -9,13 +9,13 @@ import (
 	"github.com/reviewdog/reviewdog/proto/rdf"
 )
 
-// `path` to `position`(Lnum for new file) to comment `body`s
+// `path` to `position`(Lnum for new file) to comment `body` or `finterprint`
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>comment on exported type PostedComments should be of the form "PostedComments ..." (with optional leading article)